src/pages/Register.css

```diff
@@ -0,0 +1,39 @@
+.box1{
+    display: grid;
+    place-items: center;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10%;
```

Comment:
you should use rem for margins, and no more than 4rem: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead

Reply:
for margin here, I used it for verticaly center my div In the page, so I don't know how replace it

Reply:
try flexbox/grid :)

Reply:
anyway, we don't need margin anymore here, so conflicts are resolved :)